templates/Edit.mediawiki

```diff
@@ -88,6 +92,9 @@
             |
             <h3>Create new page</h3>
             }}
+            <div id="edit-time">
+                <small name="edit-time">Last Edited {{{edit-date-string}}}</small>
```

Comment:
Sadly, we shouldn't be doing this. `getmtime` gives the last modified time of the local file. But, a git clone, has all the `mtime` set to the date of clone, not to the date of actual change. In result, this will present misleading information.

We could add something like this, but it would require a bit more work. And I don't think that belongs in this PR in that case :)

Reply:
In that case, we will probably need to go the route of my initial thought for GIT backends and compare hashes, because extracting the modified time of a file from git itself [requires a lot of plumbing](https://serverfault.com/a/401450). I can abstract the call behind the singleton.STORAGE to use mtime for local and hash for git.

Reply:
Actually, on second though: `git log -1 --format="%ad" -- path/to/file` isn't terrible. I'll try implementing it.

Reply:
I am perfectly fine with a storage backend call that returns "something unique", like `mtime` for `local`, and a hash for `git`. Just .. we won't be able to show this text on the page. Not sure that is really an issue honestly :) For any text on screen I always wonder: what do we expect a user to do with it. And .. not sure :D So I am fine with just removing this, and using the git hash etc :)

Reply:
Yeah, fair points. I removed it.